### DISCUSSION TITLE
Policy for Resource Widgets

### DISCUSSION THREAD

**Discussion by Wulfheart** [filamentphp/filament#6012] [Help]
Hello,

I have nested policies, e.g. a Deployment (`DeploymentPolicy`) has also Password on its resource page as a resource widget. Passwords are a different model and have a different policy (`PasswordPolicy`). Is it possible to apply the policy here or not? I have been unable to make it work.

**Comment by danharrin** [comment]
Policies are for models, whereas widgets aren't associated with models really. You could add methods to your policy and then use `auth()->user()->can('methodname', ModelName::class)`. Return that from `canView()` on the widget

**Reply by Wulfheart** [reply]
Ah. Thanks! Does that mean that there are such methods also for resources? 🤯

**Reply by danharrin** [reply]
Not sure what you mean...
